src/applications/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -65,6 +65,79 @@ class FacilityLocatorApp extends React.Component {
     return (
       <div className="row">
         <div className="facility-locator">
+          <div className="vads-u-margin-bottom--3 vads-u-padding--2">
```

Comment:
Was this test data that needs to be removed?

Reply:
Oh ok - we were tagged for review so I thought this PR was ready. Let us know when to take another peek! :)